### PR TITLE
[2201.6.0] Update release note with the OpenAPI changes

### PR DESCRIPTION
**Pull request by lnash94**
> Fixes #https://github.com/ballerina-platform/openapi-tools/issues/1365

## Checklist

- [ ] **Page addition**
  - [ ] Add `permalink` to pages.

- [ ] **Page removal**
  - [ ] Remove entry from corresponding left nav YAML file.
  - [ ] Add `redirect_from` on the alternative page.
  - [ ] If no alternative page, add redirection on the `redirections.js ` file.

- [ ] **Page rename**
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

- [ ] **Page restrcuture**
  - [ ] Add `permalink` to pages.
  - [ ] Add front-matter `redirect_from`.
  - [ ] Add front-matter `redirect_to:` (if applicable).

### CODE REVIEW

File: downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md [hunk 1]
```diff
@@ -84,13 +84,21 @@ Added support to provide a directory path with `bal new` to create a package in
 
 `bal init` is deprecated and will be removed in a future version. `bal new .` can be used instead.
 
+#### OpenAPI Tool
+
+- Added support for the `string` pattern with a subset of ECMA-flavored regexes that support the Ballerina regex in OpenAPI contracts using the Ballerina constraint module.
```
[the remark this answers is not in the document]
Comment (lnash94) [reply downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md:89]:
```suggestion
- Added support for openAPI regular expression templates (pattern property) defined according to [the Ballerina regular expressions grammar](https://ballerina.io/spec/lang/2022R4/#section_10.1) on string type in Ballerina client and service generation. With this support, the aforementioned regex templates will be represented as `ballerina/constraint` module annotations in the generated code.
```

[the remark this answers is not in the document]
Comment (NipunaRanasinghe) [reply downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md:89]: @lnash94 I feel like adding everything to a single sentence will make it harder to grasp the idea. shall we mention about the supported regular expression grammer as the last sentence, sepearately?

[the remark this answers is not in the document]
Comment (lnash94) [reply downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md:89]: +1

[the remark this answers is not in the document]
Comment (lnash94) [reply downloads/swan-lake-release-notes/swan-lake-2201.6.0/RELEASE_NOTE.md]: Address : https://github.com/ballerina-platform/ballerina-dev-website/pull/6960/commits/b7f68d70050eb10155561277cdd76b7e85474767

**Review by NipunaRanasinghe — APPROVED** [review]

**Pull request merged**